### PR TITLE
add-edit-delete-button

### PR DESCRIPTION
**Pull request by BogdanNegura**

### CODE REVIEW

File: src/components/layout/AddBtn.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const AddBtn = () => {
+    return (
+        <div className='fixed-action-btn'>
```
Comment (marianzburlea) [inline src/components/layout/AddBtn.js]: you're using single quotes

File: src/components/layout/AddBtn.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            <a href="#add-log-modal" className="btn-floating btn-large blue darken-2 modal-trigger">
```
Comment (marianzburlea) [inline src/components/layout/AddBtn.js]: you're using double quotes

File: src/App.js [hunk 3]
```diff
@@ -3,6 +3,9 @@ import SearchBar from './components/layout/SearchBar'
 import Logs from './components/logs/Logs'
 import AddBtn from './components/layout/AddBtn'
 import AddLogModal from './components/logs/AddLogModal'
+import EditLogModal from './components/logs/EditLogModal'
```
Comment (marianzburlea) [inline src/App.js]: Again, I see you name folders like you'd have the same restrictions JavaScript has.

File: src/components/logs/AddLogModal.js [hunk 4]
```diff
@@ -11,6 +11,10 @@ const AddLogModal = () => {
             M.toast({ html: 'Please enter a message and tech' })
         } else {
             console.log(message, tech, attention)
```
Comment (marianzburlea) [inline src/components/logs/AddLogModal.js]: when you no longer need console.log please clear it

File: src/components/techs/TechItem.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```
Comment (marianzburlea) [inline src/components/techs/TechItem.js]: I see you destructure `tech` on line 4 but you don't destructure `object` on line 2.
Is there a reason?

Comment (BogdanNegura) [reply src/components/techs/TechItem.js]: Because the "tech" is passed to PropTypes or I don't have any other explanation.

**Pull request merged**